experimenter/experimenter/features/__init__.py

```diff
@@ -80,30 +82,53 @@ def get_jsonschema(self):
 
 
 class Features:
-    _features = None
+    _features: Optional[list[Feature]] = None
+
+    @classmethod
+    def _read_manifest(
+        cls,
+        application: ApplicationConfig,
+        manifest_path: Path,
+        version: Version = None,
+    ):
+        with manifest_path.open() as manifest_file:
+            application_data = yaml.safe_load(manifest_file)
+            manifest = FeatureManifest.parse_obj(application_data)
+
+            for feature_slug, feature_model in manifest.__root__.items():
+                yield Feature(
+                    slug=feature_slug,
+                    application_slug=application.slug,
+                    model=feature_model.__root__,
+                    version=version,
+                )
 
     @classmethod
     def _load_features(cls):
         features = []
+        version_re = re.compile(r"^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)")
```

Comment:
😰 👏

Reply:
We'll have to be careful about future apps that don't conform to the full semver, this is another case where we'd wanna make sure if we really fail to match we throw to Sentry, I imagine.

Reply:
manifest-tool explicitly generates major.minor.patch versions, ignoring all the other gunk (like 0.0.0a0, 0.0.0rc1, etc.)

Reply:
Oohhhh neat 👍